### PR TITLE
GSB API response payload any data type support

### PR DESCRIPTION
**Pull request by pwalski**
Bugfix for GSB API making `payload` response field to support any data type, not only map

### CODE REVIEW

**Comment by pwalski** [comment]
I created `pre-rel-v0.13.0-rc6` to make easier testing the changes.

**Comment by pwalski** [comment]
Included test covers case of `RpcMessage` with `Item` of unit type (`()`).
Every other type of response should work too, but I did not manage to test them.
These are:
- enum https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/activity.rs#L51
- Option (special enum?) https://github.com/golemfactory/yagna/blob/e42ede5b08df1987fd526660c1604502c2dc15be/core/model/src/driver.rs#L123
- vec u8 https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/activity.rs#L206
- String https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/activity.rs#L225
- vec struct https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/activity.rs#L244
- tuple https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/appkey.rs#L120
- number https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/appkey.rs#L138
- struct (but empty - I wonder if it will be `{}` or `null`) https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/driver.rs#L269
- bool https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/driver.rs#L356
- map https://github.com/golemfactory/yagna/blob/cb756088d8e2a85ba5c6864c473d7d4c7a01a8e7/core/model/src/payment.rs#L412

**Comment by pwalski** [comment]
It was implemented outside of scope of any task, and tests cover cases we are interested in. 
Maybe we could merge it, and for additional tests we could create a task?

**Comment by pwalski** [comment]
SDK confirmed it solves the problem they faced.

File: core/gsb-api/src/api.rs [hunk 1]
```diff
@@ -255,6 +257,83 @@ mod tests {
...
+                //     "id": "0",
+                //     "payload": null,
+                // });
+                // let ws_res = serde_json::json!({
+                //     "id": "0",
+                //     "payload": {
+                //         "Ok": {},
+                //     },
+                // });
+                // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
Comment (pwalski) [inline core/gsb-api/src/api.rs]: need to delete this comment

**Review by nieznanysprawiciel — APPROVED** [review]

**Pull request merged**